packages/nodes-base/nodes/Schedule/ScheduleTrigger.node.ts

```diff
@@ -485,10 +504,26 @@ export class ScheduleTrigger implements INodeType {
 				const week = interval[i].weeksInterval as number;
 				const days = interval[i].triggerAtDay as IDataObject[];
 				const day = days.length === 0 ? '*' : (days.join(',') as string);
-				const cronTimes: ICronExpression = [minute, hour, `*/${week * 7}`, '*', day];
+				const cronTimes: ICronExpression = [minute, hour, '*', '*', day];
 				const cronExpression = cronTimes.join(' ');
-				const cronJob = new CronJob(cronExpression, executeTrigger, undefined, true, timezone);
-				cronJobs.push(cronJob);
+				if (week === 1) {
+					const cronJob = new CronJob(cronExpression, executeTrigger, undefined, true, timezone);
+					cronJobs.push(cronJob);
+				} else {
+					const cronJob = new CronJob(
+						cronExpression,
+						() => executeTrigger(true, i),
+						undefined,
+						true,
+						timezone,
+					);
+					const weeklyExecution: IDataObject = {
+						week: moment.tz(timezone).week(),
+						weekInterval: week,
+					};
+					staticData[i] = weeklyExecution;
```

Comment:
By setting the initial value to `staticData[i]` here we effectively "save" in the state that we have executed it now, preventing it from being executed.

So to clarify, let's pretend that I activate a workflow now, 10th of November, 2022 and set it to run every Friday (tomorrow would be the first execution, from the day of writing). It should run every 2 weeks.

What's going to happen is that it won't run tomorrow, since we have said it has run already, and will only run 2 weeks from now, on Nov. 25, which is not what we want.

I think we should actually unset the `staticData[i]` so it's set after the first time it actually runs.